client/packages/invoices/src/OutboundShipment/api/hooks/line/useOutboundItems.ts

```diff
@@ -14,8 +14,8 @@ export const useOutboundItems = () => {
 
     return Object.entries(
       ArrayUtils.groupBy(stockLines, line => line.item.id)
-    ).map(([itemId, lines]) => {
-      return { id: itemId, itemId, lines };
+    ).map(([_, lines]) => {
+      return lines[0]!.item;
```

Comment:
If we want the items, easier to map them directly here! Previously, were returning the invoice lines as well, but not needed :)

Ok to use `!` - to be in this array, at least 1 line must exist